admin/local/watcher/watcher_int.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: watcher-deployment
+spec:
+  selector:
+    matchLabels:
+      app: watcher
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: watcher
+    spec:
+      volumes:
+        - name: slack-webhook-url
+          secret:
+            secretName: secret-watcher
+      containers:
+        - name: watcher
+          image: qserv/lite-watcher:c1973a8b
```

Comment:
Why using the specific tag in this config? Would it be possible to rely on something like `qserv/lite-watcher:latest` (`:current`, etc.)?